pkg/registry/populator.go

```diff
@@ -94,6 +95,15 @@ func (i *DirectoryPopulator) globalSanityCheck(imagesToAdd []*ImageInput) error
 				errs = append(errs, PackageVersionAlreadyAddedErr{ErrorString: "Bundle already added that provides package and csv"})
 				break
 			}
+			if channel == image.annotationsFile.GetDefaultChannelName() {
+				containsDefault = true
+			}
+		}
+		if !containsDefault {
+			_, err := i.querier.GetBundleForChannel(context.TODO(), image.bundle.Package, image.annotationsFile.GetDefaultChannelName())
+			if err != nil {
+				errs = append(errs, EmptyDefaultChannel{ErrorString: "Cannot set an empty default channel"})
```

Comment:
If this error string isn't changing, shouldn't it be hard-coded in the `Error() string` method?

Reply:
I forgot to add `err` to the string - my bad